### DISCUSSION TITLE
Create a bucket

### DISCUSSION THREAD

**Discussion by otecmasters** [supabase/supabase#16589] [Questions] [answered]
Hi,
Is there any way to create a bucket locally with a migration created in a .sql file?

**Comment by GaryAustin1** [comment, marked as answer]
Yes.  See SQL option here:  https://supabase.com/docs/guides/storage/quickstart 
![image](https://github.com/supabase/supabase/assets/54564956/ab565260-6611-4400-8a8a-804ea0e438b5)

**Reply by otecmasters** [reply]
Thanks!
